classes/04class/exercises/c04-iac01/gepifanio/iam.tf

```diff
@@ -0,0 +1,49 @@
+resource "aws_iam_role" "s3-full-access-role" {
+    name = "s3-full-access-role"
+    assume_role_policy = <<EOF
+{
+"Version": "2012-10-17",
+"Statement": [
+{
+"Action": "sts:AssumeRole",
+"Principal": {
+"Service": "ec2.amazonaws.com"
+},
+"Effect": "Allow",
+"Sid": "EC2InstancesS3FullAccess"
+}
+]
+}
+EOF
```

Comment:
Template files might be useful for bigger policies and other config files https://registry.terraform.io/providers/hashicorp/template/latest/docs/data-sources/file